apps/src/StudioApp.js

```diff
@@ -1859,10 +1863,12 @@ StudioApp.prototype.configureDom = function (config) {
         firehoseClient.putRecord(
           'analysis-events',
           {
-            study: 'instructions-resources-tab-wip-v1',
+            study: 'instructions-resources-tab-wip-v2',
             study_group: 'under-app',
             event: 'under-app-video-click',
-            data_json: JSON.stringify([config.app, config.scriptName, config.stagePosition, config.levelPosition]),
+            script_id: config.scriptId,
```

Comment:
Would it make sense to limit recording of `under-app-video-click` to applab and gamelab here also?

Reply:
This log is nested a few layers into an if-statement which checks noInstructionsWhenCollapsed - which lets us know if we are in CSD/CSP or CSF. I believe this is sufficiently granular for what we are looking for here.